src/trio/_subprocess.py

```diff
@@ -232,10 +232,11 @@ def returncode(self) -> int | None:
     async def __aenter__(self) -> Self:
         return self
 
+    # Type ignore is for `Type of decorated function contains type "Any" ("Callable[[Process], Coroutine[Any, Any, None]]")`
```

Comment:
This feels like a mypy bug :'(

Reply:
Yea I thought so too, but I haven't made a bug report for it as of writing.

Reply:
it could be an issue in _deprecate.deprecate that didn't surface previously, but idk.